README.md

```diff
@@ -9,8 +9,9 @@ Freecoin aims to be a framework for remuneration and authentication supporting m
 [![Code Climate](https://codeclimate.com/github/Commonfare-net/macao-social-wallet.png)](https://codeclimate.com/github/Commonfare-net/macao-social-wallet)
 
 ## Design
-https://freecoin.dyne.org/images/freecoin_logo.png
-[![Freecoin Cornucopia](
+
+<img src="https://freecoin.dyne.org/images/freecoin_logo.png" width="200" alt="Freecoin Cornucopia">
+
```

Comment:
Not sure if I understand why this is needed. "Make the design image appear": was it not showing for you?

Reply:
oh sorry I see now